Makefile

```diff
@@ -159,9 +169,19 @@ thrift: idl/thrift/jaeger.thrift thrift-image
 idl/thrift/jaeger.thrift:
 	$(MAKE) idl-submodule
 
+.PHONY: idl-submodule
 idl-submodule:
 	git submodule init
 	git submodule update
 
+.PHONY: thrift-image
 thrift-image:
 	$(THRIFT) -version
+
+.PHONY: docs
+docs: $(MKDOCS_VIRTUAL_ENV)
+	bash -c 'source $(MKDOCS_VIRTUAL_ENV)/bin/activate; mkdocs serve'
```

Comment:
Should this target be serve docs?
Building the docs can be achieved by `mkdocs build`